eng/common/scripts/git-diff-changes.ps1

```diff
@@ -0,0 +1,36 @@
+<#
+  .SYNOPSIS
+  Returns git diff changes in pull request.
+
+  .DESCRIPTION
+  The script is to return diff changes in pull request.
+
+  .PARAMETER SourceCommittish
+  The branch committish PR merges from.
+  Definition of committish: https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefcommit-ishacommit-ishalsocommittish
+
+  .PARAMETER TargetCommittish
+  The branch committish PR targets to merge into.
+#>
+[CmdletBinding()]
+param (
+  [string] $SourceCommittish = "${env:BUILD_SOURCEVERSION}",
+  [string] $TargetCommittish = ("origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH}" -replace "refs/heads/")
+)
```

Comment:
We probably also need an optional parameter to pass in a path filter. (i.e. for things like eng/common).